src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenPaymentServiceProviderPort.java

```diff
@@ -177,6 +180,11 @@ public PurchaseResult authoriseKlarnaPayment(final boolean authComplete,
         return handleKlarnaAuthoriseResponse(adyenCallResult, merchantAccount);
     }
 
+    private AdyenCheckoutApiClient getAdyenCheckoutClient(final String countryCode) {
+        final AdyenCheckoutApiClient adyenCheckoutApiClient = new AdyenCheckoutApiClient(adyenConfigProperties, countryCode);
+        return adyenCheckoutApiClient;
+    }
```

Comment:
Why not just directly return `new AdyenCheckoutApiClient(adyenConfigProperties, countryCode)`

Reply:
+1

Reply:
This has been refactored in PR#6